### DISCUSSION TITLE
Autobahn vermeiden

### DISCUSSION THREAD

**Discussion by 1maxi2** [devemux86/cruiser#133] [Q&A] [answered]
<img src="https://user-images.githubusercontent.com/124886589/233773726-43a584f3-4e3b-4d16-a673-0a26fff93e6d.jpg" width="33%"/>
Ich liebe es über kleine Strassen meine Touren zu fahren. Autobahn benutze ich ganz selten.
Wenn ich Touren in Cruiser plane werde ich immer wieder über Autobahnen geleitet. Und das obwohl " Autobahn und Mautstrassen vermeiden " ausgewählt ist. Als Profil ist " Motorrad " ausgewählt. Es ist auch egal ob " schnellste oder kürzeste " ausgewählt ist.
Natürlich kann man mit vielen Wegpunkten vermeiden auf die Autobahn geleitet zu werden was umständlich ist.
Gibt es die Möglichkeit ein Profil zu installieren welches schöne kurvige  Strecken auf der Landstraße generiert ?

**Comment by devemux86** [comment, marked as answer]
The avoid options currently work with the following routing services:
- BRouter (offline)
- MapQuest
- OpenRouteService
- Valhalla

A better option may be to use customized BRouter profiles that also work offline:
https://forum.kurviger.de/t/app-offline-routing-brouter/649/19

---

Die Vermeidungsoptionen funktionieren derzeit mit den folgenden Routing-Diensten:
- BRouter (offline)
- MapQuest
- OpenRouteService
- Valhalla

Eine bessere Option kann die Verwendung von angepassten BRouter-Profilen sein, die auch offline funktionieren:
https://forum.kurviger.de/t/app-offline-routing-brouter/649/19

**Reply by 1maxi2** [reply]
Danke mit den 3 Optionen funktioniert das Routing auch ohne Autobahn 🙂
Mit Br router kenne ich mich nicht aus. Ist mir zu umständlich.  Muss man da für Br router und  Cruiser jeweils extra Karte herunterladen ?
Wird es in Cruiser einmal die Möglichkeit geben komplett offline zu planen ohne Zusatzprogramm ( Br router ) so wie es in Calimoto funktioniert ?
Wenn nicht auch kein Problem. Eine Zone mit WLAN zum planen findet sich immer irgendwo.

**Reply by devemux86** [reply]
> Muss man da für Br router und Cruiser jeweils extra Karte herunterladen ?

You need to install [BRouter app](https://play.google.com/store/apps/details?id=btools.routingapp) and download offline data for your region.
Please read the installation instructions here:

- https://github.com/devemux86/cruiser/discussions/137

---

Sie müssen [BRouter app](https://play.google.com/store/apps/details?id=btools.routingapp) installieren und Offline-Daten für Ihre Region herunterladen.
Bitte lesen Sie die Installationsanweisungen hier:

- https://github.com/devemux86/cruiser/discussions/137

**Reply by 1maxi2** [reply]
O.k. Danke

**Comment by devemux86** [comment]
Soon the route avoidance options will also work with BRouter (**offline**):
- https://github.com/devemux86/cruiser/discussions/134

---

Bald werden die Optionen zur Routenvermeidung auch mit BRouter (**offline**) funktionieren:
- https://github.com/devemux86/cruiser/discussions/134
